guides/get-going-with-gitops.mdx

```diff
@@ -0,0 +1,485 @@
+ flags:
+ - key: use-quicksort-algorithm
+   type: BOOLEAN_FLAG_TYPE
+   enabled: false
++  rollouts:
++  - segment:
++      key: internal-users
++      value: true
++segments:
++- key: internal-users
++  constraints:
++  - property: organization
++    operatior: eq
++    value: internal
++    type: STRING_COMPARISON_TYPE
+```
+
+Breaking this change down we've:
+
+#### The `internal-users` Segment
+
+```yaml features.yml
+# ...
+segments:
+  - key: internal-users
+    constraints:
+      - property: organization
+        operatior: eq
+        value: internal
+        type: STRING_COMPARISON_TYPE
+```
+
+This segment definition matches any evaluation request where there exists a key `organization` on the context, with a value `internal`.
+Remember we added the key `organization` earlier when defining the flag in code.
+We used a value derived from the request (`getOrganization(r.Context())`).
+This got the organization identifier for the calling user.
+Now, when the user happens to be associated with the `internal` organization, it will match the `internal-users` segment in Flipt.
+
+#### A New Segment Rollout Rule
+
+```yaml features.yml
+flags:
+  - key: user-quicksort-algorithm
+    # ...
+    rollouts:
+      - segment:
+          key: internal-users
+          value: true
+```
+
+Finally, we add a rollout rule to our boolean flag.
+This rule allows us to override the `enabled` property of the flag under certain conditions.
+In this instance, we're using the `segment` type rule to say when the request matches the `internal-users` segment, return the value `true`.
+
+Now our flag is configured to target internal users and enable the Quicksort algorithm for those users instead.
+
+#### Proposing and Integrating Our Change
+
+Next, we add, commit and push the change to our branch.
+
+```bash
+git add features.yml
+
+git commit -m "feat: enable use-quicksort-algorithm for internal-users"
+
+git push enable-flag-for-internal-organization
+```
+
+From here, we can open a pull-request and get feedback from our team.
+Once we've have an approval, we can merge the PR and the change goes live.
+When the change has been merged into `main`, Flipt will eventually start serving this new configuration change.
+
+We can now request our application as an authenticated user.
+Users inside the `internal` organization should get results sorted with the new Quicksort algorithm.
+Whereas, the rest of users should still be served using the old Bubble Sort algorithm.
+
+<Tip>
+  This is your opportunity to validate the change before exposing it to a wider
+  audience. Perhaps the algorithm is incorrect, maybe the order has reversed or
+  has become unstable for some entries. You can fix those changes now before
+  targeting more users.
+</Tip>
+
+### Proportional Rollout
+
+Once we're confident our change is working as expected, because we've have validated the change in production for `internal` users, we can start to roll it out to external users.
```

Comment:
```suggestion
Once we're confident our change is working as expected, since we've validated the change in production for `internal` users, we can start to roll it out to external users.
```

Reply:
This is me failing to correct all my vale suggestions everywhere